src/aggregate/aggregate_debug.h

```diff
@@ -41,6 +41,12 @@
  *         - Only applicable in FT.AGGREGATE cluster mode.
  *         - If specified, the timeout applies solely to internal shard queries,
  *           without affecting the coordinator pipeline.
+ *       - **`PAUSE_AFTER_RP_N <RP_TYPE> <N>`**:
+ *         - Inserts a pause RP **after** the first occurrence of `<RP_TYPE>`; pauses after `<N>` results
```

Comment:
1. consider mentioning that after resume, and will continue the query. Wan't clear to me at first :)
2. how are we handling timeout? maybe we should also increase timeout by the time the query was paused?